### DISCUSSION TITLE
Stereo Pose Estimation

### DISCUSSION THREAD

**Discussion by sp7412** [lagadic/visp#1376] [Q&A] [answered]
I was working through this example: https://visp-doc.inria.fr/doxygen/visp-daily/tutorial-tracking-mb-generic-stereo.html

I see this line where the pose estimates are returned separately for each camera:
```
cLeftMo, cRightMo;
tracker.getPose(cLeftMo, cRightMo);
```

Is this estimating the pose separately for each camera or estimating it from a pair of cameras? If not, can you point me to an example that estimates a single pose using a pair of stereo cameras?

Also, can you point me to some docs that explain how to interpret `cMo`?

**Comment by fspindle** [comment, marked as answer]
Estimation is done in a single process considering the pair of cameras (not each camera separately).

`cMo` is 4-by-4 [homogeneous matrix](https://visp-doc.inria.fr/doxygen/visp-daily/classvpHomogeneousMatrix.html) that gives the position/orientation of the object in the camera frame.
